env/includes/settings/tk-multi-loader2.yml

```diff
@@ -353,7 +353,7 @@ settings.tk-multi-loader2.vred:
     Jt File: [import]
     Catpart File: [import]
     Fbx File: [import]
-    VRED Scene: [load]
+    VRED Scene: [import]
```

Comment:
I'm assuming it was broken and now works? It should have been `import` all along?

Reply:
Yes that is correct it should have been import all along.